tests/keyvalue_in_memory_test.go

```diff
@@ -0,0 +1,134 @@
+package tests
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	orbitdb2 "berty.tech/go-orbit-db"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestKeyValueStoreInMemory(t *testing.T) {
```

Comment:
Can't we use the same (existing) test(s) with two backends? IMO that would be more maintainable

Reply:
We can for sure, but we need to update every test (except persistance of course) with something like table cases tests. I feels that one simple test case is enough for the moment. instead of updating each tests, IMO we should update all the tests in another PR (and remove convey by the same time 😛)

Reply:
Also tests are already taking more than 3minutes, we should try to shorten them before trying to add more of them (IMO)